### ISSUE TITLE
Minimum Fan Speed Broke syscon fan control on FAT's PS3

### ISSUE THREAD

**Issue by sin89rus** [aldostools/webMAN-MOD#490]
Latest webman mod.
Both CECHG and CECHH (the only was tested) have broken syscon now. If i change FAN Speed to SYSCON, FAN always 20%, until 85c.
How can i fix it now? I tried everything - delete webman mod / UPDATE via RECOVERY Menu / RESET to Default. Now PS3's non functional without webman fan control. 
Some bug i think, cause minimum fun speed checkbox cand be disabled, only if choose SYSCON, but this is way to death.
+++
Just tested my 2 CECHC/E - them too, with old webman mod on cfw 4.84 - WTF? How to fix it ?????

**Comment by aldostools** [comment]
Evilnat's 4.87.3 includes Cobra 8.3 with built-in fan control.

Could you explain how  Minimum Fan Speed Broke syscon fan control?

Thousands of users use webMAN without that issue.

**Comment by sin89rus** [comment]
Are you sure 1000 users tried back to SYSCON fan control?
I checked two checkboxes (auto fan control and min speed) - changed Auto control to 78c and min speed to 34%, then non restart system, change min speed to 35%, then 33%, etc. Cause min speed always 1% down at real. Maybe restart is the problem? Can i fix syscon or not? Can you test it on your console, change min speed without restart few times?

**Comment by aldostools** [comment]
Yes, thousands of HEN users use SYSCON due there is an issue in HEN that cause a black screen starting games (hypervisor panics). This issue doesn't happen on CFW because the hypervisor is patched.

The min speed only sets the minimum fan speed that can be reached by the dynamic fan control.

I suggest you that open a thread in PSX-PLACE forums asking for advice. Hardware guys can help you better with your issue, which seems to be caused by hardware, and coincided with the changes that you made in wMM settings.

**Comment by sin89rus** [comment]
4x times? Youre wrong

Dont know how its work, but i choose SYSCON, restart. Then in system infor tap to CPU/RSX few times, change settings to syscon, and now all ok. But anyway Sony's temp are so fkng hot, better use min fan speed)

**Comment by floppyD** [comment]
Have you tried unplugging your PS3 from the wall? I have noticed in the past that some of these fan settings can stick until you unplug it.

**Comment by ghost** [comment]
> Have you tried unplugging your PS3 from the wall? I have noticed in the past that some of these fan settings can stick until you unplug it.

That might be related to remote play auto on mode, i'd disable that if you don't use it.

**Comment by misterfatch** [comment]
Hi there, I had the same trouble: just reinstalled latest OFW, and got out of trouble. 
In order to reinstall, be sure to have fans on as syscon seems not to activate fans during install.
In order to do so, when wanting to go from ofw to cfw, i play a bit so that the fans blow a bit and AFTER i install my ofw or cfw;
Also when I want to go from cfw to ofw, i put maximum fan with fan control pkg 

enjoy
